mlflow/tracing/types/wrapper.py

```diff
@@ -255,7 +259,7 @@ def context(self):
         return None
```

Comment:
Should we update start_time&end_time in this class as well?

Reply:
Great catch!